plugin/src/main/java/com/denizenscript/denizen/nms/interfaces/EntityHelper.java

```diff
@@ -476,4 +477,26 @@ public CompoundTag getRawNBT(Entity entity) {
     public void modifyRawNBT(Entity entity, CompoundTag tag) {
         throw new UnsupportedOperationException();
     }
+
+    public static class EntityEntersVehicleScriptEventImpl extends EntityEntersVehicleScriptEvent {
+        @EventHandler
+        public void onEntityMount(EntityMountEvent event) {
+            fire(event, event.getMount());
+        }
+    }
+
+    public Class<? extends EntityEntersVehicleScriptEvent> getEntersVehicleEventImpl() { // TODO: once 1.20 is the minimum supported version, implement in the ScriptEvent class as usual
+        return EntityEntersVehicleScriptEventImpl.class;
+    }
+
+    public static class EntityExistsVehicleScriptEventImpl extends EntityExitsVehicleScriptEvent {
+        @EventHandler
+        public void onEntityMount(EntityDismountEvent event) {

```

Comment:
Has this been tested on older server versions? (Cause sometimes the imports explode if you have them at all)

Reply:
Tested on 1.17 and it seemed to work fine yeah - I think that was just with enums? not sure though